### DISCUSSION TITLE
The New Logo - Feedback

### DISCUSSION THREAD

**Discussion by azureitte** [ScratchAddons/ScratchAddons#289] [Discussion]
Ok, I took out the old ScratchAddons logo, but I made the new one two.

Here is the large version:
![sa-icon.png](https://user-images.githubusercontent.com/35695734/93258801-8ddea800-f7a7-11ea-9df2-f73454463508.png)

And here is the small version:
![sa-favicon.png](https://user-images.githubusercontent.com/35695734/93258899-aea6fd80-f7a7-11ea-9365-250a3a09874a.png)

I'd love to hear your feedback. Do you like the new logo? Or do you prefer the old one?

- Ucrash

**Comment by retronbv** [comment]
I like these except for the gradient, I think it should be a darker orange to the normal and be coming from a corner

**Comment by azureitte** [comment]
i wanted the gradient to be more colorful, so i made it fo from orange to pink-ish

**Comment by WorldLanguages** [comment]
I prefer the old one :P
![image](https://user-images.githubusercontent.com/17484114/93264082-ffedc780-f77c-11ea-8099-a9963940584c.png)

**Comment by GrahamSH-LLK** [comment]
Same.

**Comment by GrahamSH-LLK** [comment]
How about the old one with the gradient?

**Comment by azureitte** [comment]
The old one is asymmetrical and sloppy. I made it in 5 minutes in Scratch vector editor

**Comment by GrahamSH-LLK** [comment]
Well, I like it.

**Comment by WorldLanguages** [comment]
> The old one is asymmetrical

Huh. Nice catch. I just looked at my extension bar and all icons are symmetrical (or almost symetrical)

**Comment by jeffalo** [comment]
i actually quite like the new logo

**Comment by GrahamSH-LLK** [comment]
The old one is also almost symmetrical.

**Comment by Explosion-Scratch** [comment]
Can I try making one?

**Comment by jeffalo** [comment]
> Can I try making one?

i like this logo can we make it the logo

**Comment by Explosion-Scratch** [comment]
Alright. I made like 5, I personally like #1 and #4 best.
![Version 1](https://user-images.githubusercontent.com/61319150/93525791-28500000-f8fc-11ea-8af2-99b862d3f6b5.png)
![Version 2](https://user-images.githubusercontent.com/61319150/93525795-28e89680-f8fc-11ea-9db4-0a01cfaa5e5f.png)
![Version 3](https://user-images.githubusercontent.com/61319150/93525796-28e89680-f8fc-11ea-9b58-66eb7873e1bf.png)
![Version 5](https://user-images.githubusercontent.com/61319150/93525797-28e89680-f8fc-11ea-8d18-99c9d46b0813.png)
![Version 4](https://user-images.githubusercontent.com/61319150/93525798-29812d00-f8fc-11ea-8e85-46ed7b3e8fce.png)

**Comment by Explosion-Scratch** [comment]
Hold on, leme try with the gradients.

**Comment by Explosion-Scratch** [comment]
![Version 7](https://user-images.githubusercontent.com/61319150/93526737-7c0f1900-f8fd-11ea-9789-7ab4e3c59387.png)
![Version 6](https://user-images.githubusercontent.com/61319150/93526740-7ca7af80-f8fd-11ea-9603-d47a49e82ae9.png)
![Version 8](https://user-images.githubusercontent.com/61319150/93526741-7ca7af80-f8fd-11ea-9f92-d6691ed92278.png)

**Comment by Explosion-Scratch** [comment]
Oh, and @JeffaloBob, I just edited out the glitchy email thing of mine that you quoted btw.

**Comment by WorldLanguages** [comment]
Yeah, the issue is that we must remember in an "average" full HD screen the logo is only 32x32 in the extension bar. It's not a very good idea to have gradients and/or two icons there, since there's not much space, and the gradient might not look that well with so few possible variations of color.

**Comment by Explosion-Scratch** [comment]
> Yeah, the issue is that we must remember in an "average" full HD screen the logo is only 32x32 in the extension bar. It's not a very good idea to have gradients and/or two icons there, since there's not much space, and the gradient might not look that well with so few possible variations of color.

I see. The last one maybe? Also the last two of the first post are fine in that matter. Another alternative is that we could use one of those as the large image on stuff like the website.

**Comment by Explosion-Scratch** [comment]
> Yeah, the issue is that we must remember in an "average" full HD screen the logo is only 32x32 in the extension bar. It's not a very good idea to have gradients and/or two icons there, since there's not much space, and the gradient might not look that well with so few possible variations of color.

Just looking through my 20 extensions right now I realized that the ones with the most vibrant colors (there were 5 in particular) caught my eye the most. 2 of those had gradients.

**Comment by azureitte** [comment]
> I see. The last one maybe? Also the last two of the first post are fine in that matter. Another alternative is that we could use one of those as the large image on stuff like the website.

yeah, I actually intended for the large icon to be used on stuff like the website, organization icon, etc. And for the small icon to be used in the extension itself.

**Comment by WorldLanguages** [comment]
Closed in favor of #305
